coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
   "result": {
-    "covered_percent": 94.81
+    "covered_percent": 94.79
```

Comment:
I looked, and I'm actually not sure why. None of the < 100% covered files are touched in this PR.